dogen/schema/kwalify_schema.yaml

```diff
@@ -52,12 +52,18 @@ map:
             map:
               repo: { type: str, required: True }
               branch: { type: str, required: True }
+          cct:
+            map:
+              user: { type: str }
+              version: { type: str }
+              verbose: { type: bool }
   scripts:
     seq:
       - map:
          package: {type: str}
          exec: {type: str}
          user: {type: text}
+  cct: {type: any}
```

Comment:
becuase its CCT domain - dogen isnt strictly bound to specific CCT version - so we cannot be sure about format here. it was same before - when it will be merged to one tool we can validate here.